hidden_message.py

```diff
@@ -0,0 +1,91 @@
+# Hidden Message
+# by Cesare Tidler
+
+"""
+Encryption Key:
+    A = 1
+    B = 2
+    C = 3
+    D = 4
+    E = 5
+    F = 6
+    G = 7
+    H = 8
+    I = 9
+    J = 0
+    K = Z
+    L = Y
+"""
+
+# Get user message
+message = str(input("Enter your message to encrypt here: "))
+
+
+# Replace the characters with encrypted characters    
+string_to_build = '' # create an empty string that we'll build with characters from the message
+for letter in message: # loops over each character in the message    
+
+    # encryption key
+    if letter.upper() == "A":
```

Comment:
As stated before, if you do this, A or a will both be converted to 1, then afterwards if I want to "Decrypt" that message, the 1 will be converted back to A.  What if I want it to maintain lower case?  Just saying you might not want to force it to upper case.

Reply:
I changed the encryption key so that I don't use numbers, and made it work for each case of each letter